### DISCUSSION TITLE
reference/docs/nvm

### DISCUSSION THREAD

**Discussion by giscus** [jaywcjlove/reference#622] [Q&A]
# reference/docs/nvm

https://wangchujiang.com/reference/docs/nvm.html

**Comment by Hooligan1209** [comment]
切换使用指定版本
Mac 中通过 nvm 长期有效的切换 node 版本可能需要两个指令：

1，nvm use XXX（只对当前终端窗口有效）

2，nvm alias default XXX（长期有效，即使关闭终端窗口后，再打开一个新的终端窗口）
